docs/cloud/features/scheduler/scheduler.md

```diff
@@ -165,20 +165,42 @@ This gives you complete control over data security and network access while stil
 
 Self-hosted executors as the name indicates are self-hosted workers that take on the responsibility of "executing" changes to the data warehouse. While Tobiko Cloud schedules and plans changes, the executors are responsible for executing those changes. Executors are docker containers that are configured to connect to Tobiko Cloud as well as your data warehouse. Connected to both, the executor pulls work from the cloud, whether it's a plan or scheduled background work from a run, and execute it on your data warehouse.
 
+### Deployment Options
+
+You can deploy the executor containers using any method that works for your infrastructure and operational requirements. The executors are standard Docker containers that can be deployed in any container environment as long as they're properly configured with the required environment variables.
+
+For your convenience, we provide two reference implementations:
+
+1. **Kubernetes with Helm Chart**: For production environments, we provide a [Helm chart](./hybrid_executors_helm.md) that includes robust configurability, secret management, and scaling options.
+
+2. **Docker Compose**: For simpler environments or testing, we offer a [Docker Compose setup](./hybrid_executors_docker_compose) to quickly deploy executors on any machine with Docker.
+
+You're free to adapt these reference implementations or create your own deployment method that fits your specific needs. The only requirement is that two executor instances must be running (one for run operations and one for apply operations) and properly configured.
+
 ### Configuration
 
```

Comment:
include a config.py example like this based on your mock examples.

https://github.com/sungchun12/tcloud_hybrid_demo/blob/main/config.yaml

Reply:
Added python config that matches the YAML config and then had them both use the gateway name "gateway_a" which is common in these docs.

Also added "Network Configuration" section saying what needs to be done if behind a private network and listed the IPs.